packages/@react-spectrum/toast/src/ToastContainer.tsx

```diff
@@ -116,15 +110,24 @@ export function ToastContainer(props: SpectrumToastContainerProps): ReactElement
     return (
       <Toaster state={state} {...props}>
         <ol className={classNames(toastContainerStyles, 'spectrum-ToastContainer-list')}>
-          {state.visibleToasts.slice().reverse().map((toast) => (
-            <li
-              key={toast.key}
-              className={classNames(toastContainerStyles, 'spectrum-ToastContainer-listitem')}>
-              <Toast
-                toast={toast}
-                state={state} />
-            </li>
-          ))}
+          {state.visibleToasts.slice().reverse().map((toast) => {
+            let placements = props.placement?.split(' ');
+            let placement = placements?.[placements.length - 1] || 'bottom';
```

Comment:
can move this up and out so it's only performed once per render, could memo it too